src/bundle/Resources/public/js/scripts/core/dropdown.js

```diff
@@ -250,6 +260,30 @@
             }
         }
 
+        filterItems(event) {
+            const allItems = this.itemsListContainer.querySelectorAll('[data-filter-value]');
+
+            if (event.currentTarget.value.length < 3) {
```

Comment:
It would be good to assign this condition to some variable to give it a description.